cnudie/iter.py

```diff
@@ -3,13 +3,24 @@
 import enum
 
 import cnudie.retrieve
-import dso.labels
 import ocm
 
 
+@dataclasses.dataclass(frozen=True)
+class ExtraComponentReference:
```

Comment:
do ECR and RCRL need to be frozen (esp: hashable)? If not, we might use lists and would not need dacite to cast..